src/styles/global.css

```
@@ -1,7 +1,15 @@
 html {
-  -webkit-text-size-adjust​: ​none​;
+  -webkit-text-size-adjust: none;
+  text-size-adjust: none;
   width: 100%;
   margin: 0;
+  font-family:
+    'Pretendard',
```

Stylelint (17.14.0)

[error] Expected no quotes around "Pretendard" (font-family-name-quotes)

(font-family-name-quotes)